telebot/__init__.py

```diff
@@ -5255,7 +5249,18 @@ def reply_to(self, message: types.Message, text: str, **kwargs) -> types.Message
         :return: On success, the sent Message is returned.
         :rtype: :class:`telebot.types.Message`
         """
-        return self.send_message(message.chat.id, text, reply_to_message_id=message.message_id, **kwargs)
+        if kwargs:
+            reply_parameters = kwargs.pop("reply_parameters", None)
+        else:
+            reply_parameters = None
+        if not reply_parameters:
+            reply_parameters = types.ReplyParameters(message.message_id)
+
+        if "allow_sending_without_reply" in kwargs:
```

Comment:
why don't we check if allow_sending_without_reply is already set?

Reply:
"Optional	Pass" - you mean here? I see, will fix.

Reply:
+